examples/experimental/trial/imagenet_nas_arch_pytorch/distributed_darts.yaml

```diff
@@ -31,40 +37,52 @@ data:
 #    container_path: /mnt/data
 #    read_only: false
 
-min_validation_period: 10
+min_validation_period: 5
 
 hyperparameters:
   num_classes: 1000
   learning_rate: 0.5
   momentum: 0.9
   weight_decay: 3e-5
+  drop_path_prob: 0.2
+  drop_prob: 0.0
   label_smoothing_rate: 0.1
-  cosine_annealing_epochs: 300
+  ema_decay: 0.999
+  clip_gradients_l2_norm: 5
+  # Choices include linear, efficientnet, and cosine
+  lr_scheduler: linear
+  lr_epochs: 300
+  warmup_epochs: 5
+  # These HPs only used for efficientnet scheduler
+  lr_gamma: 0.96
+  lr_decay_every: 3
+
   init_channels: 48
   layers: 14
-  # Set global_batch_size to 1024
-  # if not using randomly generated data.
   # Batch size may be too big for GPU memory depending
   # on your GPU and number of slots_per_trial.
-  global_batch_size: 1024
+  global_batch_size: 6120
   auxiliary: false
   auxiliary_weight: 0.4
-  autoaugment: true
+  randaugment: true
   cutout: true
   cutout_length: 16
   do_SE: true
   activation: swish 
 
 resources:
-  # We recommend 8 slots with V100
-  # and at least 16 slots with K80 GPUs.
-  slots_per_trial: 8
+  slots_per_trial: 72
   shm_size: 30000000000
 
+checkpoint_storage:
+  save_experiment_best: 5
+  save_trial_best: 2
+  save_trial_latest: 1
+
 searcher:
   name: single
   metric: top1_accuracy 
-  max_steps: 300 # 1.2 million images in training set of imagenet
+  max_steps: 1000 # 1.2 million images in training set of imagenet
```

Comment:
why the difference in max_steps between const and dtrain?

Reply:
this is to train to convergence.  const uses fake data.